sse-gateway/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "sse-gateway-client-itest",
+  "version": "0.0.21",
+  "description": "Integration tests for Client API for the Jenkins SSE Gateway plugin.",
+  "files": [
+    "src/test/js"
+  ],
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jenkinsci/sse-gateway-plugin.git"
+  },
+  "keywords": [
+    "Jenkins",
+    "SSE"
+  ],
+  "author": "Tom Fennelly <tom.fennelly@gmail.com> (https://github.com/tfennelly)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://issues.jenkins-ci.org/browse/JENKINS/component/21477"
+  },
+  "homepage": "https://github.com/jenkinsci/sse-gateway-plugin",
+  "dependencies": {
+    "@jenkins-cd/sse-gateway": "0.0.21"
```

Comment:
Maybe you can use the relative dep for this + as a `devDependency` e.g. same as in `sse-gateway-sample/package.json`.

```
"@jenkins-cd/sse-gateway": "../"
```

Reply:
With that though, you might want to get the maven `clean` task to wipe `node_modules/@jenkins-cd/sse-gateway`.

Reply:
Done and done.  Not sure how sse-gateway-sample worked before, I assume it had a stale copy of sse-gateway already present.